Add SQS blogpost (#102)
metalbear-co/metalbear.co: content/en/blog/sqs-splitting/index.md

---
title: "Split Queues To Share Cloud Development Environments"
subtitle: "How multiple developers can debug the same event driven microservice in one shared environment, and split 
queue messages between them"
description: "Introducing SQS splitting in mirrord for Teams."
lead: "Introducing SQS splitting in mirrord for Teams."
tags:
  - devtools
  - mirrord 
  - SQS
  - cloud
images: []
date: 2024-10-01T06:00:00+00:00
lastmod: 2024-10-01T06:00:00+00:00
draft: false
weight: 50
contributors: ["Tal Zwick"]
---

One of the reasons [mirrord for Teams](https://mirrord.dev/docs/overview/teams/) is a gamechanger for many 
organizations is that it makes it possible for a whole team to work with one shared cloud environment. They don’t 
need to run the whole environment for each developer, they don’t need to have their entire architecture running on 

> their laptops, and they do not need to redeploy to Kubernetes to test new code or debug it. Developers just press 

Reviewer: ```suggestion
their laptops, and they don't need to redeploy to Kubernetes to test new code or debug it. Developers just press 
```
